string channel_name

time interval_start
time interval_end

time cache_start
time cache_end

uint32 num_samples
float64[] ranges

uint8 stationary